docs/doc/60-contributing/03-rfcs/20220728-metasrv-txn.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Transaction support in MetaService
+description: 
+  Transaction support in MetaService
+---
+
+MetaService support transaction with `KVApi`, transaction parameters are:
+
+* condition: `condition` is a list of predicates. If all of them success, the `if_then` will be executed, otherwise `else_then` op will be executed.
+* if_then: `if_then` is a list of operations will be executed when all condition evaluates to true.
+* else_then: `else_then` is a list of operations will be executed when not all condition evaluates to true.
```

Comment:
```suggestion
* condition: `condition` is a list of predicates. If all of them succeed, the `if_then` will be executed, otherwise `else_then` op will be executed.
* if_then: `if_then` is a list of operations that will be executed when all condition evaluates to be true.
* else_then: `else_then` is a list of operations that will be executed when not all condition evaluates to be true.
```